packages/oas-to-snippet/package.json

```diff
@@ -50,7 +50,7 @@
     "test": "vitest run --coverage"
   },
   "dependencies": {
-    "@readme/httpsnippet": "^9.0.0",
+    "@readme/httpsnippet": "^10.0.0",
```

Comment:
🥺 can you bump this to 10.0.1 that i just published with [that swift change](https://github.com/readmeio/httpsnippet/pull/226)?

Reply:
https://github.com/readmeio/oas/pull/859/commits/17cf2bf4d0accfeb652a431844b3e753b99423b5